spec/daemon_logger_spec.rb

```diff
@@ -112,27 +112,29 @@
     lambda { subject.level = 'unknown' }.should raise_error(ArgumentError)
   end
 
-  it 'rotation' do
-    log = DaemonLogger.new("tmp/se1.log", level: 'trace', log_rotate_age: 3, log_rotate_size: 10000)
-    # 100 bytes
-    log.warn "test1"*20
-    File.exist?("tmp/se1.log").should == true
-    File.exist?("tmp/se1.log.0").should == false
-
-    # 10000 bytes
-    100.times { log.warn "test2"*20 }
-    File.exist?("tmp/se1.log").should == true
-    File.exist?("tmp/se1.log.0").should == true
-    File.read("tmp/se1.log.0") =~ /test2$/
-
-    # 10000 bytes
-    100.times { log.warn "test3"*20 }
-    File.exist?("tmp/se1.log").should == true
-    File.exist?("tmp/se1.log.1").should == true
-    File.exist?("tmp/se1.log.2").should == false
-
-    log.warn "test4"*20
-    File.read("tmp/se1.log.0") =~ /test5$/
+  unless ServerEngine.windows?
```

Comment:
Why is this test omitted for windows environment? Just for path concatenation, or any other reasons?
It's better to use `pending "reason..."` to show the test skipped if there are any reason to be skipped.